### ISSUE TITLE
Make StrictTranslation property public

### ISSUE THREAD

**Issue by igitur** [OfficeDev/Open-XML-SDK#711]
# Before submitting an issue, please fill this out

Is this a:
- [X] Issue with the OpenXml library
- [ ] Question on library usage

**Description**

Please provide a simple description of the issue encountered.

**Information**

- .NET Target: All
- Latest commit on `master`

In https://github.com/OfficeDev/Open-XML-SDK/blob/0f1764a866e36d615e22d247e780fb7675046e9a/src/DocumentFormat.OpenXml/Packaging/OpenXmlPackage.cs#L240 `StrictTranslation` is internal. 

Would it be possible to make this property public (at least the `get`)?

Knowing whether a package was saved in strict or non-strict mode eases the subsequent parsing. For example, in strict mode, dates are stored in `yyyy-MM-dd` format and in non-strict the serial number is stored.

**Comment by twsouthwick** [comment]
I'm ok with that. @tomjebo any issue with making this public?

We're getting ready for a 2.11 release and can probably squeeze this in.

**Comment by twsouthwick** [comment]
@igitur Can you prepare a PR for this?

**Comment by igitur** [comment]
@twsouthwick Sure. Maybe one more thing. Is `StrictTranslation` the correct name to expose publicly? I understand its meaning in the internal sense where you're translating namespaces from strict to non-strict. But maybe for a public property something like `IsStrictMode` is more applicable. Your call.

**Comment by twsouthwick** [comment]
Good catch. Definately want `Is...` naming . I like your idea. Feel free to rename it while making it public

**Comment by twsouthwick** [comment]
@tomjebo Thoughts?

**Comment by tomjebo** [comment]
A couple things, I agree with the IsStrict... part of the naming, however, I'm not sure that IsStrictMode would be accurate. The assumption here is that the StrictTranslation field set to "true" indicates that the workbook will contain dates in the yyyy-MM-dd format while "false" indicates they will use the serial number format. I can't say that will always be the case as StrictTranslation simply means that at least 1 relationship was found with "http://purl.oclc.org..." (it's not specific about which one). It's not a bad assumption that this would indicate that **Excel** saved it in Strict conformance mode (if Excel was the last application to write to it). However, this scheme may fail if there are mixed strict and transitional URI's used in the relationship parts of a workbook, as is seen in the example ("StrictXlsx.xlsx") from [Loading xlsx saved in strict mode...](https://github.com/ClosedXML/ClosedXML/issues/1378).

`  <Relationship Id="rId3" Type="http://purl.oclc.org/ooxml/officeDocument/relationships/extendedProperties" Target="docProps/app.xml"/>
  <Relationship Id="rId2" Type="http://schemas.openxmlformats.org/package/2006/relationships/metadata/core-properties" Target="docProps/core.xml"/>
  <Relationship Id="rId1" Type="http://purl.oclc.org/ooxml/officeDocument/relationships/officeDocument" Target="xl/workbook.xml"/>
`

In that case, you might assume you can't process the workbook because it was saved as strict by Excel but that assumption might be wrong.

Bottom line is that I would be more comfortable exposing a get only property named something like "IsStrictTranslation" or "StrictRelationshipFound" or something. Just to be truthful in advertising this property.

**Comment by igitur** [comment]
Agree. I like `StrictRelationshipFound`. It's very explicit about its intention. I'll prepare a PR based on that if you concur.
